DatasetsInfo.md

```diff
@@ -0,0 +1,117 @@
+# Dataset information
+
+- most important info: Period a.k.a. orbit time (time in which the satellite makes 1 circle around the Earth)
+
+# Copernicus
+
+## Sentinel-1
+
+2 satellites: Sentinel-1A, Sentinel-1B
+
+- 1 satellite:
+  - Orbital cycle: 12 days
+  - Orbits per cycle: 175
+  - Calculated time for 1 orbit: ~98.742 min (should be the same as Period)
+  - Period (time to circle the Earth 1 time = 1 orbit): 98.6 min
+  - Revisit time (of the same area) on the equator: 12 days
+
+- Both satellites:
+  - Period: 98.6 min / 2 = ‭49.3 min = 2958000 ms **(used in `datasets.ts`)**
```

Comment:
I would argue that  I'm not sure if many end users will read this - probably more programmers will do (and probably the easiest way to compare `Date` objects is to use `getTime()` method which returns the time in milliseconds).

But because the numbers are smaller in minutes, the probability of making a mistake is smaller, so I changed that to minutes as you suggested.

Reply:
Yes, sorry if I my intent wasn't communicated clearly - it was mostly code readability (for example, ability for programmers reading code to spot mistakes, like z missing `0`) that I had in mind.

Looks good, thx, resolving!